File: app/src/main/java/com/pixels/blockies/app/draws/ViewContext.java
package com.pixels.blockies.app.draws;

/**
 * Created by mark on 16.11.14.
 */
public class ViewContext {
    private int border = -1;
    private int thickness = -1;
    private float width = -1;
    private float height = -1;
    private int blockHeight = -1;
    private int blockWidth =  -1;

    public int getBorder() {
        return border;
    }

    public void setBorder(int border) {
        this.border = border;
    }

    public int getThickness() {
        return thickness;
    }

    public void setStrokeThickness(int thickness) {
        this.thickness = thickness;
    }

    public float getWidth() {
        return width;
    }

    public void setWidth(float width) {
        this.width = width;
    }

    public float getHeight() {
        return height;
    }

    public void setHeight(float height) {
        this.height = height;
    }

    public int getBlockHeight() {
        return blockHeight;
    }

    public void setBlockHeight(int blockHeight) {
        this.blockHeight = blockHeight;
    }

    public int getBlockWidth() {
        return blockWidth;
    }

    public void setBlockWidth(int blockWidth) {
        this.blockWidth = blockWidth;
    }
}

File: app/src/main/java/com/pixels/blockies/game/draws/ViewContext.kt
package com.pixels.blockies.game.draws

/**
 * The context that holds important information
 * in order to draw all the block correctly
 */
class ViewContext {
    private var border = -1
    private var thickness = -1
    private var width = -1f
    private var height = -1f
    private var blockHeight = -1
    private var blockWidth = -1
    fun getBorder(): Int {
        return border
    }

    fun setBorder(border: Int) {
        this.border = border
    }

    fun getThickness(): Int {
        return thickness
    }

    fun setStrokeThickness(thickness: Int) {
        this.thickness = thickness
    }

    fun getWidth(): Float {
        return width
    }

    fun setWidth(width: Float) {
        this.width = width
    }

    fun getHeight(): Float {
        return height
    }

    fun setHeight(height: Float) {
        this.height = height
    }

    fun getBlockHeight(): Int {
        return blockHeight
    }

    fun setBlockHeight(blockHeight: Int) {
        this.blockHeight = blockHeight
    }

    fun getBlockWidth(): Int {
        return blockWidth
    }

    fun setBlockWidth(blockWidth: Int) {
        this.blockWidth = blockWidth
    }
}